visualization/app/codeCharta/ui/mapTreeView/mapTreeView.level.component.html

```diff
@@ -11,6 +10,7 @@
 		ng-mouseleave="$ctrl.onMouseLeave()"
 		ng-right-click="$ctrl.openNodeContextMenu($event)"
 		ng-click="$ctrl.onClickNode()"
+		id="{{ ::$ctrl.node.path }}"
```

Comment:
I hope this does not change the event propagation in a way that other parts are highlighted / active / similar due to the id change?

Reply:
No, not at all. The id here is not used in our production code. It's only there to access the tree-structure easily.

Introduced here:
https://github.com/MaibornWolff/codecharta/pull/1100/files#diff-58b5703126f177e3766594a6099259ac